examples/node/lib/rest/api/v2010/account/call/feedbackCallSummary.ts

```diff
@@ -226,7 +228,7 @@ export class FeedbackCallSummaryInstance {
   sid?: string | null;
   testString?: string | null;
   testInteger?: number | null;
-  testObject?: object | null;
+  testObject?: PhoneNumberCapabilities | null;
```

Comment:
Need to update one of the unit tests to assert on one of these updated properties to make sure the types are accessible.

Reply:
Added